src/Microsoft.DotNet.Build.Tasks/Targets/tests.targets

```diff
@@ -21,10 +21,12 @@
 
   <!-- General xunit options -->
   <PropertyGroup>
-    <XunitHost>corerun.exe</XunitHost>
-    <TestHost>$(XunitHost)</TestHost>
+    <UseTestHost Condition="'$(UseTestHost)' == ''">true</UseTestHost>
```

Comment:
So are you assuming that this will be set to false when the test itself it an exe that will run on the full desktop framework and doesn't need a specific host?